docs/usage/tutorials.md

```diff
@@ -0,0 +1,589 @@
+Contamination can come from many places, typically from the the host of a host-associated sample, however this can also come from laboratory processing samples.
+A common contaminant in Illumina sequencing is a spike-in control of the genome of PhiX virus, which we can download with the following command.
+
+```bash
+curl -O https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/819/615/GCF_000819615.1_ViralProj14015/GCF_000819615.1_ViralProj14015_genomic.fna.gz
+```
+
+### Preparing Input
+
+#### Sample sheet
+
+You provide the sequencing data FASTQ files to nf-core/taxprofiler via a input 'sample sheet' `.csv` file.
+This is a 6 column table, that includes sample and library names, instrument platform, and paths to the sequencing data.
+
+Open a text editor, and create a file called `samplesheet.csv`.
+Copy and paste the following csv file into the file and save it.
+
+```csv
+sample,run_accession,instrument_platform,fastq_1,fastq_2,fasta
+ERX5474932,ERR5766176,ILLUMINA,ERX5474932_ERR5766176_1.fastq.gz,ERX5474932_ERR5766176_2.fastq.gz,
+ERX5474932,ERR5766176_B,ILLUMINA,ERX5474932_ERR5766176_B_1.fastq.gz,ERX5474932_ERR5766176_B_2.fastq.gz,
+```
+
+Here we have specified two libraries of the same sample, that they were sequencing on Illumina platforms, and the paths to the FASTQ files.
+If you had placed your FASTQ files elsewhere, you would give the full path (i.e., with relevant directories) to the `fastq_1`, `fastq_2` and `fasta` columns.
+
+#### Database sheet
+
+For the database(s), you also supply these via a `.csv` file.
+This 4 column table contains the tool the database has been built for, a database name, the parameters you wish reads to be quiered against the given database with, and a path to a `.tar.gz` archive file or a directory containing the database files.
+
+Open a text editor, and create a file called `database.csv`.
+Copy and paste the following csv file into the file and save it.
+
+```csv
+tool,db_name,db_params,db_path
+kraken2,db1,--quick,testdb-kraken2.tar.gz
+centrifuge,db2,,test-db-centrifuge.tar.gz
+centrifuge,db2_trimmed,--trim5 2 --trim3 2,test-db-centrifuge.tar.gz
+kaiju,db3,,kaiju/
+```
+
+You can see here we have specified the Centrifuge database twice, to allow comparison of different settings.
+Note that the each database of the same tool has a unique name.
+Furthermore, while the Kraken2 and Centrifuge databases have been supplied as `.tar.gz` archives, the Kaiju database has been supplied as a directory.
+
+### Running the pipeline
+
+Now that we have the sequencing reads (in FASTQ format), the databases (directory or .tar.gz), and a reference genome (FASTA, optionally gzipped), we can now run them with the pipeline. The following command will perform short read quality control, remove contaminant reads, merge multiple libraries for each sample, run the three profilers, and finally generate standardised profiles.
+
+```bash
+nextflow run nf-core/taxprofiler -r 1.1.0 -profile docker \
```

Comment:
do you mean the version **1.0.1**?

Reply:
Nope, I mean 1.1.0 which is why  `taxpasta merge` failed :grimacing: 

I tested with `dev` which will be 1.1.0